src/librustdoc/html/static/main.js

```diff
@@ -360,31 +360,69 @@
                 }
             }
 
+            function nbElements(obj) {
+                var size = 0, key;
+                for (key in obj) {
+                    if (obj.hasOwnProperty(key)) {
+                        size += 1;
+                    }
+                }
+                return size;
+            }
+
+            function extractGenerics(val) {
+                var values = val.substring(val.indexOf('<') + 1, val.lastIndexOf('>'));
+                return values.split(/\s*,\s*/);
```

Comment:
Would it be prudent to also include the "base name" in this? It looks like this throws away the "Result" in `Result<File, Error>`. From how this is used it seems like it wouldn't be included in Levenstein comparisons because of that.

Reply:
That's the point. :)